src/integration_sdk.js

```diff
@@ -28,8 +28,8 @@ const defaultSdkConfig = {
   endpoints: [],
   adapter: null,
   version: 'v1',
-  httpAgent: new http.Agent({ keepAlive: true }),
-  httpsAgent: new https.Agent({ keepAlive: true }),
+  httpAgent: new http.Agent({ keepAlive: true, maxSockets: 10, maxTotalSockets: 10 }),
+  httpsAgent: new https.Agent({ keepAlive: true, maxSockets: 10, maxTotalSockets: 10 }),
```

Comment:
Wouldn't defining just `maxSockets` be sufficient, as the SDK is always talking to only one host, the Integration API? Or then just use `maxTotalSockets` and only communicate that in the docs. I think only using `maxSockets` is fine.

Reply:
Yeah, it's not really needed to set `maxTotalSockets`. I put it there "just for safety", but can also just remove it. If later the SDK can talk to the Asset Delivery API, too, that setting might matter, but also it's fine to live with just the per-host limit.